deploy_to_live.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
Line comment worked.

It's only ever possible to deploy from {dev to live} or from {live to test} with Terminus, so these names are shorter.

Unless we also want a deploy_dev_to_live.sh that just calls those two.